sketch.js

```diff
@@ -92,18 +94,30 @@ function setupSliderLabel(sliderX, sliderY, verticalFlag, label){
     labelY = labelY * 2;
   }
 
+
   // Wrap text if the ratio of text width to canvas width is too big ( >= .08)
   if((textWidth(label) / width) >= (2 / 25)) {
     var labelLines = label.split(" ");
     var modifier = 0;
     
     for(var line in labelLines) {
-      text(labelLines[line], labelX, labelY + modifier);
+      var lineWidth = textWidth(labelLines[line]);
+      var centeredX = labelX - floor(lineWidth / 2);
+      
+      textL = createP(labelLines[line]);
```

Comment:
This variable is never initialized so you're creating an incidental global.

Also, how about `labelText` instead...